cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,46 @@
 /// <reference types='cypress' />
+import PageObject from '../support/PageObject';
+import OrderFormPageObject from '../support/pages/orderForm.pageObject';
+import HomeAndCataloguePageObject
+  from '../support/pages/homeCatalogue.pageObject';
+import { faker } from '@faker-js/faker';
+
+const orderForm = new OrderFormPageObject();
+const homePage = new HomeAndCataloguePageObject();
+const allertMessage = new PageObject();
+
+const testData = {
+  name: faker.person.firstName(),
+  country: faker.location.country(),
+  city: faker.location.city(),
+  creditCard: faker.finance.creditCardNumber(),
+  month: '07',
+  year: '2026',
+  successMessage: 'Thanks you for your purchase!'
```

Comment:
There is a typo in the success message string. It should be 'Thank you for your purchase!' instead of 'Thanks you for your purchase!'.